routing/router.go

```diff
@@ -335,6 +336,12 @@ type Config struct {
 	// returned.
 	QueryBandwidth func(edge *channeldb.ChannelEdgeInfo) lnwire.MilliSatoshi
 
+	// UpdateConnRequests checks if there are currently any connection
+	// requests to a node with the given pubKey. If there are and if the
+	// nodes net addresses have changed then the old connection requests
```

Comment:
Something I miss here is that the comment says that the old connection request are cancelled, but the implementation doesn't do that?